src/components/molecules/select/index.tsx

```diff
@@ -318,72 +316,63 @@ const SSelect = React.forwardRef(
               <ArrowDownIcon size={16} />
             </div>
           )}
-          <CacheProvider
-            value={createCache({
-              key: "my-select-cache",
-              prepend: true,
-            })}
-          >
-            {
-              <GetSelect
-                isCreatable={isCreatable}
-                searchBackend={filterOptions}
-                options={
-                  hasSelectAll && isMultiSelect
-                    ? [{ value: "all", label: "Select All" }, ...options]
-                    : options
+          {
```

Comment:
comment: Looks more dangerous than it is. Simply removing the `CacheProvider` from Emotion